imagebuildah/stage_executor.go

```diff
@@ -1799,6 +1799,20 @@ func (s *StageExecutor) intermediateImageExists(ctx context.Context, currNode *p
 		}
 	}
 	for _, image := range images {
+		// If `--cache-ttl` was specified then
+		// short early if following image is
```

Comment:
`if --cache-ttl was specified then ignore image if it was crated before the specified duration`